scripts/ci/conda/setup.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+conda update -n base -c defaults conda -y
+conda install conda-build ninja compilers -y
+
+pwd
+ls
+git clone https://github.com/conda-forge/proj.4-feedstock.git
+
+cd proj.4-feedstock
+cat > recipe/recipe_clobber.yaml <<EOL
+source:
+  git_url: https://github.com/OSGeo/PROJ.git
+  git_rev: ${GITHUB_SHA}
+  url:
+  sha256:
+
+build:
+  number: 2112
```

Comment:
why 2112 ?

Reply:
> why 2112 ?

@warmerdam honoring 😄  More importantly, if someone is actually using these packages, we don't want them to clash with normally-installed ones from @conda-forge that would have build numbers incrementing from 0.